apps/codeforafrica/src/lib/api.netlify-cms/getOffices.js

```diff
@@ -0,0 +1,25 @@
+import { join } from "path";
+
+import getCollectionData from "./getCollectionData";
+
+const officesDir = join(process.cwd(), "content/offices");
+
+export default function getOffices(fields) {
+  const offices = getCollectionData(officesDir, fields);
+  return offices.map((office) => {
+    return {
+      map: {
+        center: {
+          lat: JSON.parse(office.map.latitude),
```

Comment:
1. Why not `Number.parseFloat`?
2. Why not parse once and reuse in both `center` and `position` rather than parsing _twice_?